lib/index.js

```diff
@@ -205,7 +205,10 @@ function createPrefix(loaderContext, globalVarTree, globalVars, useEval) {
 module.exports = function loader(originalSource, inputSourceMap) {
     const self = this;
     const callback = this.async();
-    let source = originalSource;
+    let source = originalSource
+        //.replace(/\/\/.*/g, '') // remove line comments (somehow this bugs)
```

Comment:
Seems like this would strip everything in the source file after the first comment. You probably only want to match until EOL.